### DISCUSSION TITLE
Question about how to extract/download all files of all formats from any website.

### DISCUSSION THREAD

**Discussion by Leonardo-2** [ckan/ckan#8080] [Q&A]
Hello everybody! Please, I would like a lot of help from you:

1. Is it possible to download all data/files of all formats from any CKAN website? If it is possible, what would it be like?

2. How to download all CKAN site files from JSON files with site-wide metadata?

**Comment by pdelboca** [comment]
Hello @Leonardo-2 !

Check `ckanapi` package. It has a `ckanapi dump` command that allows to download all datasets and files from an instance:

 - https://github.com/ckan/ckanapi
 - https://github.com/ckan/ckanapi/blob/master/README.md#bulk-dumping-and-loading
